deployment/README.md

```diff
@@ -1,88 +1,34 @@
 # CCNP Deployment Guide
 
-CCNP is designed for collecting confidential computing primitives in cloud native environments. It's designed to run as DaemonSet (for Kubernetes cluster) or container (for docker container) on confidential virtual machines nodes, such as Intel Trust Domain (TD). Below diagram illustrates CCNP deployment process. In this document, it will use Intel TD as an example of CVM and deploy CCNP on Intel TD nodes.
-
-![Deployment diagram](../docs/ccnp-deployment-process.png)
-
+CCNP is designed for collecting confidential computing primitives in cloud native environments. It can run as DaemonSet in a Kubernetes cluster or containers in a Docker environment on confidential virtual machines, such as Intel TDX guest(TD). 
 
 ## Create TD
 
-You can use [cvm image rewriter](../tools/cvm-image-rewriter/README.md) to prepare a TD enlightened guest image.
+Before moving forward, please make sure you have a TD booted. The CCNP service installation will happen in the TD.
+
+You can use [cvm image rewriter](../../tools/cvm-image-rewriter/README.md) to prepare a TD enlightened guest image.
 
 **NOTE:**
- - If your initial guest image already has TDX kernel, it's required to run [plugin](../tools/cvm-image-rewriter/pre-stage/) 08 and 09 to set device access permission.
- - If your initial guest image is a normal Ubuntu guest image, it's required to run [plugin](../tools/cvm-image-rewriter/pre-stage/) 05 to install TDX kernel and then 08, 09 to set device access permission.
+ - If your initial guest image already has TDX kernel, it's required to run [plugin](../../tools/cvm-image-rewriter/pre-stage/) 08 and 09 to set device access permission.
+ - If your initial guest image is a normal Ubuntu guest image, it's required to run [plugin](../../tools/cvm-image-rewriter/pre-stage/) 05 to install TDX kernel and then 08, 09 to set device access permission.
```

Comment:
Same as above.

Reply:
Let's update it after your PR is merged.